docs/RFCS/00000000_template.md

```diff
@@ -51,6 +52,11 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
 (You may replace the section title if the intent stays clear.)
 
 - Its interaction with other features is clear.
+- It covers where this feature may be surfaced in other areas of the product
+   - What metrics should be measured? Should they be surfaced in the admin UI?
+   - How will we debug issues? Do these metrics need to be surfaced anywhere?
+   - Does the user interaction with this feature map to our existing design
+     paradigms?
```

Comment:
What are our "existing design paradigms"? @kuanluo do we have a style guide (not just for the brand, but for the UX) anywhere?

Reply:
Using the word "design" here is so ambiguous as to be unproductive / destructive about what needs to happen. Also, no questions!

May I suggest instead:

"
If the change influences a user-facing interface, make sure to preserve consistent user experience (UX). Prefer to avoid UX changes altogether unless the RFC also argues for a clear UX benefit to users. If UX has to change, then prefer changes that match tbe UX for related features, to give a clear impression to users of homogeneous CLI / GUI elements. Avoid UX surprises at all costs. If in doubt, ask for input from other engineers with past UX design experience and from your design department.
"